packages/dd-trace/src/debugger/devtools_client/state.js

```diff
@@ -24,12 +24,23 @@ module.exports = {
    * To fix this, specify a more unique file path, e.g `demo-project1/index.js` instead of `index.js`
    *
    * @param {string} path
-   * @returns {[string, string] | undefined}
+   * @returns {[string, string, string | undefined] | undefined}
```

Comment:
Unrelated to this PR, but the JSDoc return value was out of date, so fixed it while I was here